src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/DefaultEditorTemplates.cs

```diff
@@ -245,6 +248,7 @@ public static IHtmlContent ObjectTemplate(IHtmlHelper htmlHelper)
             }
 
             var serviceProvider = htmlHelper.ViewContext.HttpContext.RequestServices;
+            var htmlEncoder = serviceProvider.GetRequiredService<HtmlEncoder>();
```

Comment:
Talked to @dougbu ; we'll copy the so-called "Null Encoder" from Razor as a private class here. And perhaps give it a more correct name: something like `PassThroughHtmlEncoder` or something.